packages/cli/src/storybook/storybookUtils.ts

```diff
@@ -5,6 +5,26 @@ const frameworkToPackageMap = {
   [StorybookFramework.REACT]: '@storybook/react',
 };
 
+const frameworkToLoadOptionsMap = {
+  [StorybookFramework.REACT]: '@storybook/react/dist/server/options',
+};
+
+export function getStorybookLoadConfig() {
```

Comment:
can we rename this to `getStorybookLoadOptions` as thats how its called inside sotrybook everywhere, also fits better to `getSbOption` as a name